packages/react/src/components/form/CharacterCounter.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import utilityClasses from '../../utils/utility.module.css';
+import { ErrorMessage } from '../Typography';
+
+export type CharacterLimitProps = Omit<
+  CharacterCounterProps,
+  'id' | 'value' | 'size'
+>;
+
+type CharacterCounterProps = {
+  /** The message indicating the remaining character limit. */
+  label?: (count: number) => string;
+  /** The description of the maximum character limit for screen readers. */
+  srLabel?: string;
```

Comment:
Nå ser jeg at dette var her fra før, men jeg er usikker på "srLabel" er et beskrivende nok navn for dette. Skjermlesere er jo bare et eksempel på verktøy som benytter `aria-describedby`-attributtet. I praksis brukes det kanskje ikke til så mange andre formål (som jeg vet om i hvert fall), men i utgangspunktet er det jo der for å gjøre dom-en lesbar for alle typer verktøy. Hva med "description" eller noe i de baner?